.github/dependabot.yml

```
@@ -14,6 +14,12 @@ updates:
     kubernetes:
       patterns:
       - "k8s.io/*"
+    ginkgo:
+      patterns:
+      - "github.com/onsi/ginkgo/v2"
+    gomega:
+      patterns:
+      - "github.com/onsi/gomega"
```

I was expecting something like this

```suggestion
    ginkgo:
      patterns:
      - "github.com/onsi/ginkgo/v2"
      - "github.com/onsi/gomega"
```

Why are they split?

---

I split them since they don't often see releases at the same time.  This split configuration would have dependabot update them separately rather than combine them.